package.json

```diff
@@ -85,7 +85,7 @@
     "glob": "^7.1.2",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.8.0",
-    "lerna": "^2.1.2",
+    "lerna": "^3.15.0",
```

Comment:
3.16.1 was released recently, lets set that as the lower bound.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2535/commits/653d751d7ad2331e1e96bdd3a131237b83b4e096.